src/main/java/cz/smarteon/loxone/app/LoxoneApp.java

```diff
@@ -120,4 +137,36 @@ public <T extends Control> T getControl(final @NotNull String name, final @NotNu
         }
         return found;
     }
+
+    @JsonIgnore
+    @SuppressWarnings("unchecked")
+    @Nullable
+    public <T extends Control> List<T> getControlsForRoom(final @NotNull Room room) {
```

Comment:
1. the type parameter `<T>` seems to be useless - it will always return the mix of controls, so simple `List<Control>` is enough
2. kindly document the method is not simple getter but computes the result everytime